app/scripts/lib/errors.js

```diff
@@ -100,15 +100,25 @@ define([
      * @param {String} [context]
      */
     toError: function (type, context) {
-      var message = this.toMessage(type);
-      if (! message) {
```

Comment:
Thanks @TDA - [toMessage](https://github.com/mozilla/fxa-content-server/blob/9ec6b1812e76fa9576462bd9c4cebf4544698f9f/app/scripts/lib/errors.js#L35) is guaranteed to return a string, even if no type was passed in. Line 105 was the lone line in this file to not be covered by test coverage, and I could find no way to cause it to be covered w/o mocking in `toMessage` to return a falsey value.